examples/p4est_2d_dgsem/elixir_navierstokes_NACA0012airfoil_mach08.jl

```diff
@@ -130,12 +130,26 @@ lift_coefficient = AnalysisSurfaceIntegral(semi, force_boundary_names,
                                                                    u_inf(equations),
```

Comment:
Should these be `uinf`, `linf`, `rhoinf` like in the callback file? Or maybe the callback file could be made like what we have here?

Reply:
Hm I do not think that it hurts that we have the underscore here and not in the source file.